### PR TITLE
Fix url for username with space

### PR DESCRIPTION
**Pull request by ppigazzini**
Fix the html error:
"Bad value for attribute href on element a:
Illegal character in path segment: space is not allowed."

### CODE REVIEW

File: server/fishtest/templates/users.mak [hunk 1]
```diff
@@ -79,7 +83,7 @@
             <td class="text-end">${int(user['games_per_hour'])}</td>
             <td class="text-end">${int(user['cpu_hours'])}</td>
             <td class="text-end">${int(user['games'])}</td>
-            <td class="text-end"><a href="/tests/user/${user['username']}">${user['tests']}</a></td>
+            <td class="text-end"><a href="/tests/user/${urllib.parse.quote(user['username'])}">${user['tests']}</a></td>
```
[the remark this answers is not in the document]
Comment (ppigazzini) [reply server/fishtest/templates/users.mak:86]: Few:
```
Error: Bad value /tests/user/Andrew Grant for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 2059, column 34; to line 2059, column 68
text-end"><a href="/tests/user/Andrew Grant">511</a

Error: Bad value /tests/user/Thomas A. Anderson for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 2455, column 34; to line 2455, column 74
text-end"><a href="/tests/user/Thomas A. Anderson">0</a><

Error: Bad value /tests/user/Ron Britvich for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 2716, column 34; to line 2716, column 68
text-end"><a href="/tests/user/Ron Britvich">60</a>

Error: Bad value /tests/user/[AMD Users] NeoGen for attribute href on element a: Illegal character in path segment: [ is not allowed.
From line 4588, column 34; to line 4588, column 74
text-end"><a href="/tests/user/[AMD Users] NeoGen">0</a><

Error: Bad value /tests/user/Werner j. for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 6082, column 34; to line 6082, column 65
text-end"><a href="/tests/user/Werner j.">0</a><

Error: Bad value /tests/user/Sriram Girish for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 6721, column 34; to line 6721, column 69
text-end"><a href="/tests/user/Sriram Girish">0</a><

Error: Bad value /tests/user/Dual E5-2660 v3 for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 9142, column 34; to line 9142, column 71
text-end"><a href="/tests/user/Dual E5-2660 v3">0</a><

Error: Bad value /tests/user/Sriram Girish - The Jd for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 10411, column 34; to line 10411, column 78
text-end"><a href="/tests/user/Sriram Girish - The Jd">0</a><

Error: Bad value /tests/user/samuel H. for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 10438, column 34; to line 10438, column 65
text-end"><a href="/tests/user/samuel H.">0</a><

Error: Bad value /tests/user/Samuel H. for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 11158, column 34; to line 11158, column 65
text-end"><a href="/tests/user/Samuel H.">0</a><

Error: Bad value /tests/user/alan r for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 16774, column 34; to line 16774, column 62
text-end"><a href="/tests/user/alan r">0</a><

Error: Bad value /tests/user/Ivan Ivec for attribute href on element a: Illegal character in path segment: space is not allowed.
From line 17089, column 34; to line 17089, column 65
text-end"><a href="/tests/user/Ivan Ivec">22</a>
```

**Comment by ppigazzini** [comment]
@peregrineshahin 
DEV updated, see here the other errors to fix in "users" pages
ttps://validator.w3.org/nu/?doc=https%3A%2F%2Fdfts-0.pigazzini.it%2Fusers

**Comment by ppigazzini** [comment]
@vdbergh if I'm not wrong `urllib` is not used in `actions.mak` so it can be dropped.

**Comment by vdbergh** [comment]
> @vdbergh if I'm not wrong `urllib` is not used in `actions.mak` so it can be dropped in this PR. Noted while checking the format of all mako templates.

Yes it is not used. Cfr the above review.

**Pull request merged**